.gitignore

```diff
@@ -6,5 +6,6 @@ coverage
 npm-debug.log
 
 */**/yarn.lock
+*/**/package-lock.json
```

Comment:
The new version of NPM, `npm 5`, automatically generates a lock file on install. We should discuss at some point if we want to throw all these lock files into `.gitignore`, including `yarn.lock` or pick one to keep.